_format/1.0/index.md

```diff
@@ -82,7 +82,7 @@ A document **MUST** contain at least one of the following top-level members:
 * `meta`: a [meta object][meta] that contains non-standard
   meta-information.
 
-The members `data` and `errors` **MUST NOT** coexist in the same document.
+The member's `data` and `errors` **MUST NOT** coexist in the same document.
```

Comment:
This change is incorrect. The `data` and `errors` do not belong to "the member", they ARE members themselves.

Reply:
My mistake, I have reverted this change